runtime/compiler/ilgen/Walker.cpp

```diff
@@ -4955,7 +4948,13 @@ break
        return NULL;
        }
 
-    int32_t constToLoad = 0;
+   int32_t constToLoad = 0;
+   if(symbol->getRecognizedMethod() == TR::com_ibm_jit_JITHelpers_supportsIntrinsicCaseConversion)
```

Comment:
`constToLoad` is no longer used and can be removed. Also add space after `if`

Reply:
changed according to comments. thanks!